website/views/options/social/challenges.jade

```diff
@@ -91,6 +91,19 @@ script(type='text/ng-template', id='partials/options.social.challenges.html')
             label
               input(type='radio', name='search-participation-radio', ng-click='search._isMember = undefined', checked='checked')
               =env.t('either')
+          h4=env.t('challengedOwnedFilterHeader')
+          .radio
+            label
+              input(type='radio', name='search-participation-radio', ng-click='search._isOwner = true')
+              =env.t('challengedOwnedFilter')
+          .radio
+            label
+              input(type='radio', name='search-participation-radio', ng-click='search._isOwner = false')
+              =env.t('challengedNotOwnedFilter')
+          .radio
+            label
+              input(type='radio', name='search-participation-radio', ng-click='search._isOwner = undefined')
```

Comment:
Why? You can own and not be a member of a challenge.
On Apr 11, 2015 2:31 PM, "Blade Barringer" <notifications@github.com> wrote:

> In website/views/options/social/challenges.jade
> <https://github.com/HabitRPG/habitrpg/pull/5015#discussion_r28198582>:
>
> > @@ -91,6 +91,19 @@ script(type='text/ng-template', id='partials/options.social.challenges.html')
> >              label
> >                input(type='radio', name='search-participation-radio', ng-click='search._isMember = undefined', checked='checked')
> >                =env.t('either')
> > +          h4=env.t('challengedOwnedFilterHeader')
> > +          .radio
> > +            label
> > +              input(type='radio', name='search-participation-radio', ng-click='search._isOwner = true')
> > +              =env.t('challengedOwnedFilter')
> > +          .radio
> > +            label
> > +              input(type='radio', name='search-participation-radio', ng-click='search._isOwner = false')
> > +              =env.t('challengedNotOwnedFilter')
> > +          .radio
> > +            label
> > +              input(type='radio', name='search-participation-radio', ng-click='search._isOwner = undefined')
>
> Oh, I see, you can't filter by membership AND ownership. I guess that
> makes sense.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/5015/files#r28198582>.
>